src/views/partials/labels.handlebars

```diff
@@ -0,0 +1,18 @@
+<h3>Labels</h3>
+{{!-- List Labels --}}
+<form>
```

Comment:
currently this is a PATCH to /boards/card/labels/{boardId}/{cardId} in the routes. might want to shove that link as the action="" of this form, so you have it when doing client side.

Reply:
Added that to the form